buildSrc/src/main/kotlin/plugin-conventions.gradle.kts

```diff
@@ -6,15 +6,13 @@ plugins {
 
 java {
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(8))
```

Comment:
I don't want to go all the way to 17 just yet. Do we need to upgrade Java at all? The higher the version, the lower the compatibility. If we need to, can we upgrade to 11?